internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/config/QualysScannerConfiguration.java

```diff
@@ -0,0 +1,93 @@
+/*
+ *
+ *   Copyright (c) 2019, WSO2 Inc., WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ * /
+ */
+
+package org.wso2.security.tools.scanmanager.scanners.qualys.config;
+
+import org.wso2.security.tools.scanmanager.common.config.YAMLConfigurationReader;
+
+/**
+ * Qualys Scanner Specific Configuration.
+ */
+public class QualysScannerConfiguration extends YAMLConfigurationReader {
+
+    private static final QualysScannerConfiguration INSTANCE = new QualysScannerConfiguration();
+
+    public QualysScannerConfiguration() {
```

Comment:
This is a singleton class. Therefore this constructor should be a private one to prevent others from instantiating this class.
```suggestion
    private QualysScannerConfiguration() {
```